src/main/scala/collections/Steps.scala

```diff
@@ -0,0 +1,12 @@
+package collections
+
+object Steps extends App {
+  val N = 10
+  println(countSteps(N).size)
```

Comment:
here you check the size of list and get 14 for 10-length list. Isn't it odd? Not mentioning the result sequence is full of 1s